### PR TITLE
[MMI] Add confirm add custodian token component unit test

### PR DESCRIPTION
**Pull request by albertolive**
## Explanation

Add confirm add custodian token component unit test

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [x] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [x] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [x] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by NidhiKJha — DISMISSED** [review]

File: ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.test.js [hunk 1]
```diff
@@ -114,4 +119,120 @@ describe('Confirm Add Custodian Token', () => {
 
     expect(screen.getByTestId('error-message')).toBeVisible();
   });
+
+  it('clicks the cancel button and removes the connect request', async () => {
+    renderWithProvider(<ConfirmAddCustodianToken />, store);
+
+    await waitFor(() => {
+      const cancelButton = screen.getByTestId('cancel-btn');
+      fireEvent.click(cancelButton);
```
Comment (DDDDDanica) [inline ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.test.js:128]: We can move this out of await?

File: ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js [hunk 2]
```diff
@@ -56,6 +58,71 @@ const ConfirmAddCustodianToken = () => {
...
+              ? 'Custodian RPC confirm'
+              : 'Custodian RPC cancel',
+            custodian: connectRequest.custodian,
+            apiUrl: connectRequest.apiUrl,
+          },
+        });
+
+        if (isConfirm) {
+          history.push(CUSTODY_ACCOUNT_ROUTE);
+        }
```
Comment (DDDDDanica) [inline ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js:116]: Missing setLoading false at the end of it?

File: ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.test.js [hunk 3]
```diff
@@ -114,4 +119,120 @@ describe('Confirm Add Custodian Token', () => {
...
+    const customStore = configureMockStore()(customMockedStore);
+
+    renderWithProvider(<ConfirmAddCustodianToken />, customStore);
+
+    await waitFor(() => {
+      const confirmButton = screen.getByTestId('confirm-btn');
+      fireEvent.click(confirmButton);
+    });
+
+    expect(screen.getByTestId('pulse-loader')).toBeInTheDocument();
```
Comment (DDDDDanica) [inline ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.test.js:236]: I think you can combine this test with ones before to save the tests :))

File: ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.test.js (hunk 1 above)
Comment (albertolive) [reply ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.test.js:128]: Yes!

File: ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js (hunk 2 above)
Comment (albertolive) [reply ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js:116]: Fixed

**Review by pedronfigueiredo — APPROVED** [review]

**Review by NidhiKJha — APPROVED** [review]

**Pull request merged**